content/04.results.md

```diff
@@ -189,16 +187,17 @@ The prompt for Methods (Figure @fig:ai_revision) is more focused in keeping the
 Original text is on the left and suggested revision on the right.
 ](images/diffs/methods/phenoplier-paragraph-01.svg "Diffs - PhenoPLIER methods paragraph 01"){#fig:methods:phenoplier width="100%"}
 
-We revised a paragraph in PhenoPLIER that contains two numbered equations (Figure @fig:methods:phenoplier).
-The revised text contains very few changes: all the equations, citations and most of the original text was preserved.
-However, we found it remarkable how the model identified a mistake in the original text (line 8) and fixed it in the revision (line 7).
-Indeed, the equation with the univariate model used by PrediXcan (lines 4 to 6 in the original) includes the *true* effect size $\gamma_l$ (`\gamma_l`) instead of the *estimated* one $\hat{\gamma}_l$ (`\hat{\gamma}_l`).
+We revised a paragraph in PhenoPLIER that contained two numbered equations (Figure @fig:methods:phenoplier).
+The model made very few changes, and all the equations, citations, and most of the original text were preserved.
+However, we found it remarkable how the model identified a wrong reference to a mathematical symbol (line 8) and fixed it in the revision (line 7).
+Indeed, the equation with the univariate model used by PrediXcan (lines 4-6 in the original) includes the *true* effect size $\gamma_l$ (`\gamma_l`) instead of the *estimated* one $\hat{\gamma}_l$ (`\hat{\gamma}_l`).
 
 
 In PhenoPLIER, we found one large paragraph with several equations that the model failed to revise, although it performed relatively well in revising the rest of the section.
 In CCC, the revision of this section was good overall, with some minor and easy-to-fix issues as in the other sections.
 
 
 We also observed issues from revising one paragraph at a time without context.
-For instance, in PhenoPLIER, one of the first original paragraphs in this section mentions the linear models used by S-PrediXcan and S-MultiXcan, but without providing any equations or details.
-The details about these models, including the equations, are presented in the following paragraphs, but since the model have not seen that yet, it opted to add those equations right away (in the correct Manubot/Markdown format).
+For instance, in PhenoPLIER, one of the first paragraphs mentions the linear models used by S-PrediXcan and S-MultiXcan, but without providing any equations or details.
+These were presented in the following paragraphs, but since the model have not seen that yet, it opted to add those equations right away (in the correct Manubot/Markdown format).
```

Comment:
```suggestion
These were presented in the following paragraphs, but since the model had not encountered that yet, it opted to add those equations immediately (in the correct Manubot/Markdown format).
```